### ISSUE TITLE
t/60-cidrvalidate.t and t/90-misc.t started to fail

### ISSUE THREAD

**Issue by eserte** [atoomic/Net-Whois-IANA#14]
The test suite fails on my smoker systems:
```
# Failed test 'cidr'
# at t/60-cidrvalidate.t line 29.
# +----------------+----+--------------+
# | GOT            | OP | CHECK        |
# +----------------+----+--------------+
# | 191.96.58.0/24 | eq | 191.96.58/24 |
# +----------------+----+--------------+
# Seeded srand with seed '20201011' from local date.
t/60-cidrvalidate.t .... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/4 subtests 
This doesn't look like start-end
 at /usr/home/cpansand/.cpan/build/2020101109/Net-Whois-IANA-0.45-0/blib/lib/Net/Whois/IANA.pm line 596.
# Looks like your test exited with 255 just after 8.
t/90-misc.t ............ 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 1/9 subtests 
```
Looking at http://fast-matrix.cpantesters.org/?dist=Net-Whois-IANA;reports=1#sl=7,1 it seems that the failures started in August 2020.

**Comment by atoomic** [comment]
Thanks @eserte I can also reproduce it... this is coming from an upstream module...
we should either handle the two flavors or remove this very fragile unit test

**Comment by atoomic** [comment]
this should be fixed with the release of v0.46
